core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -69,7 +69,9 @@ object ApiVersion {
     "0.11.0-IV1" -> KAFKA_0_11_0_IV1,
     // Introduced leader epoch fetches to the replica fetcher via KIP-101
     "0.11.0-IV2" -> KAFKA_0_11_0_IV2,
-    "0.11.0" -> KAFKA_0_11_0_IV2
+    "0.11.0" -> KAFKA_0_11_0_IV2,
+    // Introduced LeaderAndIsrRequest V1, UpdateMetadataRequest V4 and MetadataResponse V5 via KIP-112
+    "0.11.1-IV0" -> KAFKA_0_11_1_IV0
```

Comment:
@becketqin Sure. I have updated the patch to address this issue. And the patch has been rebased onto the latest trunk.